app/src/main/java/de/westnordost/streetcomplete/quests/drinking_water/AddDrinkingWater.kt

```diff
@@ -22,6 +22,7 @@ class AddDrinkingWater : OsmFilterQuestType<DrinkingWater>() {
         and (!seasonal or seasonal = no)
         and (!disused or disused = no)
         and (!ruins or ruins = no)
+        and (!intermittent or intermittent = no)
```

Comment:
maybe grouping it with seasonal and put it into next line after it would make sense?

or is it ordered in some other way?

Reply:
I don't know, I just added it at the end. I changed it to follow your suggestion.